sonaranalyzer-dotnet/src/Rspec2Resx.targets

```diff
@@ -3,23 +3,28 @@
   <PropertyGroup>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
+
   <ItemGroup Condition="'$(Rspec2Resx)' != ''">
     <RSpecFiles Include="$(ProjectDir)..\..\rspec\$(Rspec2Resx)\*.html"/>
     <RSpecFiles Include="$(ProjectDir)..\..\rspec\$(Rspec2Resx)\*.json"/>
     <!--Must be explicitely added, because we are executed after the list has already been filled by what exists in the folder -->
     <EmbeddedResource Include="RspecStrings.resx" />
   </ItemGroup>
+
+  <!-- BeforeTargets="DispatchToInnerBuilds" is needed since we target multiple .net frameworks and the RspecStrings.resx file should be gennerated only once. //-->
```

Comment:
```suggestion
  <!-- BeforeTargets="DispatchToInnerBuilds" is needed since we target multiple .net frameworks and the RspecStrings.resx file should be generated only once. //-->
```